src/lib/TextExtraction.js

```diff
@@ -79,9 +79,14 @@ class TextExtraction {
       }
     });
 
+    var children = text;
+    if (props.renderText && typeof props.renderText === 'function') {
```

Comment:
I think we need to remove `renderText` from the props otherwise it will probably trigger some propType warnings